src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5120,4 +5120,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_TypeReserved" xml:space="preserve">
     <value>The type name '{0}' is reserved to be used by the compiler.</value>
   </data>
-</root>
+  <data name="ERR_RefExtensionMustBeValueTypeOrConstrainedToOne" xml:space="preserve">
+    <value>The receiver of the reference extension method '{0}' must be a value type or a generic type constrained to one.</value>
+  </data>
+  <data name="ERR_RefReadOnlyExtensionMustBeValueType" xml:space="preserve">
+    <value>The receiver of the readonly reference extension method '{0}' must be a value type.</value>
```

Comment:
>receiver [](start = 15, length = 8)

Is this error reported on a declaration? If that is the case, perhaps we should say "this parameter" instead of "receiver".